workflow-service/src/test/java/com/redhat/parodos/workflow/version/WorkFlowVersionServiceImplTest.java

```diff
@@ -24,7 +24,7 @@ public void testGetHash() {
 		// when
 		assertDoesNotThrow(() -> {
 			String hash = this.workFlowVersionService.getHash(object);
-			assertEquals(hash, "b0c8ed039dc102c0bab6a5e979931a0b");
+			assertEquals(hash, "3ffe85fac846ce265bcef0534e6334b7");
```

Comment:
This change won't be needed after https://github.com/parodos-dev/parodos/pull/239